### DISCUSSION TITLE
Success and Fail callbacks called for task independent of task status

### DISCUSSION THREAD

**Discussion by durgadeshi** [apache/airflow#29783] [Q&A]
I created a DAG and Task using @dag and @task annotations but success and fail callbacks are calling every time independent of status of executed task. Logs or print statements in the call back methods also not printing in the UI log.

DAG callbacks also not working. No method is calling.

**Comment by potiuk** [comment]
It has very little details for anyone to help you -but if you have a reproducible case (including sharing your DAG code examples and logs) , specify the Airflow wersion in a new issue, it would increase your chance of getting help.
